### PR TITLE
[hot fix] manually changed companion to hellohacks 2024

### PR DESCRIPTION
**Pull request by AllanT102**
🎟️ Ticket(s): Closes #

👷 Changes: A brief summary of what changes were introduced.

💭 Notes: Any additional things to take into consideration.

Wait! Before you merge, have you checked the following:

📷 Screenshots

(prefer animated gif)

## Checklist

- [ ] Looks good on large screens
- [ ] Looks good on mobile

### CODE REVIEW

File: src/pages/public/RegistrationForms/MembershipForm/index.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ const MembershipFormContainer = (props) => {
       fname: first_name,
       lname: last_name,
       major: memberType === "UBC" || memberType === "UNI" ? major : "",
-      email: email.toLowerCase(),
```
Comment (AllanT102) [inline src/pages/public/RegistrationForms/MembershipForm/index.js:211]: @voctory  this is good to go into prod right?

Comment (voctory) [reply src/pages/public/RegistrationForms/MembershipForm/index.js:211]: Didn't get to test it on dev branch yet

Comment (voctory) [reply src/pages/public/RegistrationForms/MembershipForm/index.js:211]: Should be fine for prod

File: src/pages/public/Companion/events/index.js [hunk 2]
```diff
@@ -390,5 +390,69 @@ export default [
...
       }]
     }
+  },
+  {
+    /* Date indicating when event ends, this also indicates which app the companion app will render, to be safe, but a couple days after event ends */
+    activeUntil: new Date(new Date("2024-10-10").getTime() + (7 * 24 * 60 * 60 * 1000)),
+    /* id of event in dynamodb, used for queries */
+    eventID: "hello-hacks",
+    /* year of event in dynamodb, used for queries */
+    year: 2023,
```
Comment (voctory) [inline src/pages/public/Companion/events/index.js:400]: Should this be 2024?

File: src/pages/public/Companion/events/index.js [hunk 3]
```diff
@@ -390,5 +390,69 @@ export default [
...
+    /* options defining params for the companionLayout */
+    options: {
+      /* Biztech logo for event */
+      BiztechLogo: BizTechDBLogo,
+      /* Logo for event */
+      Logo: DBLogo,
+      /* Displayed title of event */
+      title: "Hello Hacks 2024",
+      /* Displayed date of event */
+      date: "Saturday, October 5th",
```
Comment (voctory) [inline src/pages/public/Companion/events/index.js:412]: Calendar says Oct 4th to Oct 6th?

File: src/pages/public/Companion/components/CompanionLayout.js [hunk 4]
```diff
@@ -138,7 +140,7 @@ const CompanionLayout = (params) => {
       landing
     },
     email,
-    setEmail,
+    // setEmail,
```
Comment (voctory) [inline src/pages/public/Companion/components/CompanionLayout.js:143]: Why are we commenting setEmail?

**Review by voctory — CHANGES_REQUESTED** [review]

File: src/pages/public/Companion/components/CompanionLayout.js (hunk 4 above)
Comment (AllanT102) [reply src/pages/public/Companion/components/CompanionLayout.js:143]: build errors – not in use for this code change

File: src/pages/public/Companion/events/index.js (hunk 2 above)
Comment (AllanT102) [reply src/pages/public/Companion/events/index.js:400]: not worrying about this

**Review by voctory — APPROVED** [review]

**Pull request merged**